roles/openshift_health_checker/openshift_checks/sdn.py

```diff
@@ -0,0 +1,354 @@
+"""
+Check that the SDN is routing traffic properly.
+"""
+
+import datetime
+import os
+
+import ipaddress
+from ansible.module_utils import six
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class SDNCheck(NotContainerizedMixin, OpenShiftCheck):
+    """A check to run relevant diagnostics on the SDN."""
+
+    name = 'sdn'
+    tags = ['health']
+
+    def is_active(self):
+        """Skip hosts that are not masters or nodes."""
+        group_names = self.get_var('group_names', default=[])
+        master_or_node = 'oo_masters_to_config' in group_names or \
+                         'oo_nodes_to_config' in group_names
+        return super(SDNCheck, self).is_active() and master_or_node
+
+    def run(self):
+        if self.want_full_results:
+            # Gather diagnostic information and perform diagnostics on a master
+            # or node host.
+            try:
+                self.save_command_output('journal',
+                                         ['/bin/journalctl', '--since=-5m'])
```

Comment:
User deployed openshift cluster and noticed issue with network (pod is not scheduled or some other error).
If the user runs this script after 5m, then we may not get any relevant info. 
Either return last 5 minutes of logs beginning from when it is logged last time or get fixed number of lines from the last (capture last 4k lines?)

Reply:
Fixed to get the last 5 minutes of logs, but it's a little icky—it requires getting the last log line, parsing its timestamp, and using that to compute the interval.